.pylintrc

```diff
@@ -0,0 +1,43 @@
+[MAIN]
+
+# Specify a score threshold under which the program will exit with error.
+fail-under=10
+
+[BASIC]
+
+# Minimum line length for functions/classes that require docstrings, shorter
+# ones are exempt.
+docstring-min-length=100
+
+
+# Good variable names which should always be accepted, separated by a comma.
+good-names=s,
+           f,
+           i,
+           j,
+           k,
+           e,
+           _
```

Comment:
This looks a bit odd. Why are separate lines needed?

Reply:
TBH I'm not that fussed. 

This is how pylint itself formats it (`pylint --generate-rcfile`) so I kept it the same. 
Usually the reason is for better Git diffs when you change options.

I'm happy to put it on one line if you prefer

Reply:
I mean it just looks very odd to me. I'd change it to one line.

Reply:
:heavy_check_mark:  Done

Reply:
Thanks.